mautic/mautic
# Email reports enhancement

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | n
| Related developer documentation PR URL | n
| Issues addressed (#s or URLs) | #990
| BC breaks? | n
| Deprecations? | n

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
PR added Send count, Read ratio, Clicks, Unique clicks, Clicks ratio, Unique clicks ratio, Unsubscribed, Unsubscribed ratio to email reports.


#### Steps to test this PR:
1. Create Email report and select from columns new options   Send count, Read ratio, Clicks, Unique clicks, Clicks ratio, Unique clicks ratio, Unsubscribed, Unsubscribed ratio
2. See report

#990
